modules/virt-preparing-container-disk-for-vms.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+[id="virt-preparing-container-disk-for-vms_{context}"]
+= Preparing a container disk for virtual machines
+
+A container disk must be built with a virtual machine image and pushed to a container registry. It can then either be imported into a DataVolume and attached to a virtual machine or attached directly to a virtual machine as an ephemeral containerDisk volume.
```

Comment:
s/containerDisk/`containerDisk`

Because this is a procedure, I think it's especially important to phrase this intro in the active voice.